astro/src/content/docs/customize/look-and-feel/_theme-upgrade.mdx

```diff
@@ -1,6 +1,16 @@
 import InlineField from 'src/components/InlineField.astro';
 import InlineUIElement from 'src/components/InlineUIElement.astro';
 
+As you upgrade your FusionAuth version, you'll also want to make sure you test and possibly upgrade your theme.
+
+### CSS
+
+If you can meet your look and feel customization needs by only modifying the CSS file, rather than the theme templates, upgrades to later versions of FusionAuth will be easier.
```

Comment:
```suggestion
If you can meet your look and feel customization needs by only modifying the CSS stylesheet, rather than the theme templates, upgrades to later versions of FusionAuth will be easier.
```